Mathlib/LinearAlgebra/Dual/Lemmas.lean

```diff
@@ -105,7 +105,7 @@ variable {R : Type uR} {M : Type uM} {K : Type uK} {V : Type uV} {ι : Type uι}
 
 section CommSemiring
 
-variable [CommSemiring R] [AddCommMonoid M] [Module R M] [DecidableEq ι]
+variable [CommSemiring R] [AddCommMonoid M] [Module R M]
```

Comment:
I'm not sure why this file compiles without any new `classical` calls.

Reply:
I checked locally: because `b` two lines below is not actually used, and `\iota` only in `section Finite`. So, I'd also remove the `variable (b : Basis ι R M)` on the next line.